pkgs/misc/emulators/pcsx2/default.nix

```diff
@@ -1,57 +1,80 @@
-{ alsaLib, cmake, fetchFromGitHub, glib, gtk2, gettext, libaio, libpng
-, makeWrapper, perl, pkgconfig, portaudio, SDL2, soundtouch, stdenv
-, wxGTK30, zlib }:
+
+
+{ alsaLib, cmake, fetchFromGitHub, glib, gettext, harfbuzz, lib, libaio
+, libpng, libpcap, libxml2, makeWrapper, perl, pkgconfig, portaudio
+, SDL2, soundtouch, stdenv, udev, wxGTK, zlib
+
+, gtk2Support ? true, gtk2
```

Comment:
Why keep the gtk2 support? Either way, GTK 3 and GTK 2 support is mutually exclusive in both WxWidgets and [pcsx2](https://github.com/PCSX2/pcsx2/blob/9c65b73ae6ae383de6c727f4ee900446f2430646/cmake/SearchForStuff.cmake#L40-L44) so you might want to have just a single argument for choosing.

Reply:
@jtojnar good point. I did some more research and it looks like `gtk3` support is still experimental. All distros I checked were sticking with `gtk2`, as recommended by upstream. I had troubles getting `pcsx2` to even build with `gtk3`. I think it would take more time than I am willing to put into this for me to figure out how to get the package to compile with `gtk3`. Sorry :disappointed: